docs/UserGuide.md

```diff
@@ -161,27 +161,40 @@ Examples:
 
 ### Editing a contact: `edit`
 
-_{To be updated...}_
+Current: Edit contacts whose names contain any of the given keywords or ids.
 
+Format: `INDEX/ID [--name NAME] [--id ID] [--phone PHONE] [--email EMAIL] [--url URL] [--address ADDRESS] [--tag TAG] [--stat STATUS] [--pos POSITION]`
```

Comment:
Error: `--stat` and `--pos` no longer exists

Missing: `--application` for editing applications